webapp/src/components/ui/MatrixView/MatrixGraphView.tsx

```diff
@@ -63,41 +63,34 @@ export default function MatrixGraphView(props: PropTypes) {
   const { data = [], columns = [], index = [] } = matrix;
   const classes = useStyles();
   const [columnName, setColumnName] = useState<string[]>([]);
-  const [monotonous, setMonotonous] = useState<boolean>(false);
+  const [monotonic, setMonotonic] = useState<boolean>(false);
 
   const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
     setColumnName(event.target.value as string[]);
   };
 
-  const monotonousChange = () => {
-    setMonotonous(!monotonous);
+  const monotonicChange = () => {
+    setMonotonic(!monotonic);
   };
 
   const unitChange = (tabBase: Array<number>) => {
-    const tab = [...tabBase];
-    for (let i = 0; i < tab.length; i += 1) {
-      if (i === 0) {
-        tab.splice(i, 1, 100 / tab.length);
-      } else {
-        tab.splice(i, 1, 100 / tab.length + tab[i - 1]);
-      }
-    }
-    return tab;
+    const stepLength = 100 / tabBase.length;
+    return tabBase.map((el, i) => stepLength * (i + 1));
   };
 
   return (
     <div className={classes.root}>
       <div className={classes.container}>
         <FormControl className={classes.form}>
-          <InputLabel className={classes.input} id="demo-multiple-chip-label">{t('data:graphSelector')}</InputLabel>
+          <InputLabel className={classes.input} id="chip-label">{t('data:graphSelector')}</InputLabel>
           <Select
             className={classes.select}
-            labelId="demo-multiple-chip-label"
-            id="demo-multiple-chip"
+            labelId="chip-label"
+            id="chip"
```

Comment:
prepend id with something specific like "matrix-graph" (in case it's in a page with another select)